interpreters/wamr/Makefile

```diff
@@ -32,22 +42,25 @@ VPATH += $(WAMR_UNPACK)/product-mini/platforms/nuttx
 
 MAINSRC = main.c
 
-PROGNAME  = iwamr
+PROGNAME  = iwasm
 PRIORITY  = $(CONFIG_INTERPRETERS_WAMR_PRIORITY)
 STACKSIZE = $(CONFIG_INTERPRETERS_WAMR_STACKSIZE)
 MODULE    = $(CONFIG_INTERPRETERS_WAMR)
 
 $(WAMR_TARBALL):
 	$(Q) echo "Downloading $(WAMR_TARBALL)"
-	$(Q) wget $(WAMR_URL)
+	$(Q) curl -O -L $(WAMR_URL)
```

Comment:
why switch to curl?

Reply:
No functional difference, just because curl does not need to be installed on some systems (like Mac)

Reply:
But other location use wget too. it's better to make a new patch to replace all wget to curl if the change can simplify the prerequest.

Reply:
Done, I'll send another patch to do that